dagger-core/src/main/java/io/odpf/dagger/core/sink/kafka/KafkaSerializationSchemaFactory.java

```diff
@@ -0,0 +1,19 @@
+package io.odpf.dagger.core.sink.kafka;
+
+import io.odpf.dagger.common.configuration.Configuration;
+import io.odpf.dagger.common.core.StencilClientOrchestrator;
+import io.odpf.dagger.common.serde.DataTypes;
+import io.odpf.dagger.core.sink.kafka.builder.KafkaJsonSerializerBuilder;
+import io.odpf.dagger.core.sink.kafka.builder.KafkaProtoSerializerBuilder;
+import io.odpf.dagger.core.utils.Constants;
+
+public class KafkaSerializationSchemaFactory {
```

Comment:
how about moving these to common as well?

Reply:
Did not move this to common now since wanted to have the source level abstract present in common now. Logic moved to common is now specific for serde logic. Also, the factories are not reusable. Wanted to keep things which can be used by others in common.